modules/getting-cluster-version-status-and-update-details.adoc

```diff
@@ -27,7 +27,7 @@ $ oc get clusterversion
 [source,terminal]
 ----
 NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
-version   4.6.4     True        False         6m25s   Cluster version is 4.6.4
+version   {product-version}.0     True        False         6m25s   Cluster version is {product-version}.0
```

Comment:
[Looks like][1] this will have to be bumped by hand:

![image](https://user-images.githubusercontent.com/209920/129967340-e9ad24a4-3dd6-4158-b00c-634179a51533.png)

[1]: https://deploy-preview-35601--osdocs.netlify.app/openshift-enterprise/latest/installing/validating-an-installation.html#getting-cluster-version-and-update-details_validating-an-installation

Reply:
If you use `[source,terminal,subs="attributes+"]`, it'll render.

Reply:
Rebased on `main` and added `subs="attributes+"` with db1330b6c -> 0b101ec52.

Reply:
[Success][1]?

![image](https://user-images.githubusercontent.com/209920/142285054-b9ba034b-dbe5-4c03-b272-2e2237b83d6f.png)

[1]: https://deploy-preview-35601--osdocs.netlify.app/openshift-enterprise/latest/installing/validating-an-installation.html#getting-cluster-version-and-update-details_validating-an-installation